build.sbt

```diff
@@ -7,15 +7,15 @@ lazy val microservice = Project("customs-service-status", file("."))
   .enablePlugins(play.sbt.PlayScala, SbtDistributablesPlugin)
   .settings(
     majorVersion := 0,
-    scalaVersion := "2.13.12",
+    scalaVersion := "3.5.0",
     ScoverageKeys.coverageExcludedFiles :=
       "<empty>;com.kenshoo.play.metrics.*;.*definition.*;prod.*;testOnlyDoNotUseInAppConf.*;" +
         "app.*;.*BuildInfo.*;.*Routes.*;.*repositories.*;.*controllers.test.*;.*services.test.*;.*metrics.*",
     ScoverageKeys.coverageMinimumStmtTotal := 80,
     ScoverageKeys.coverageFailOnMinimum := true,
     ScoverageKeys.coverageHighlighting := true,
+    ScoverageKeys.coverageEnabled := true,
```

Comment:
Does `coverageEnabled` default to true anyway? (ie. can we remove this line?) Or has this changed in Scala 3 and we now need to explicitly set it?

Reply:
yes it defaults to true, i thought perhaps there was an issue with it, but i've removed now.